int32 robot_id
geometry_msgs/Point position
